content/release-10-16-0/applications-10-16-0-bundle/10-16-0-0.md

```diff
@@ -131,17 +131,23 @@ Web SDK</td>
 </tr>
 
 <tr>
-<td></td>
+<td>Web SDK</td>
 <td>An example of custom validators usage for the device grid component is now available in the Tutorial application.</td>
 <td>MTM-41533</td>
 </tr>
 
 <tr>
-<td></td>
+<td>Web SDK</td>
 <td>Plugins now support translations, that is, translations added to plugins are now loaded to the shell application.</td>
 <td>MTM-49787</td>
 </tr>
 
+<tr>
+<td>Web SDK</td>
+<td>The HOOK_COMPONENTS now allows to lazy load the components using <code>loadComponent</code> and <code>loadConfigComponent</code>.</td>

```

Comment:
"to lazy load" -> either "to lazily load" or "lazy loading" (i prefer the first)

Reply:
I also thought about "to lazily load" but then I thought it might be a verb as is, like "to lazy-load". Will change it.